Services/DataBaseService/Startup.cs

```diff
@@ -53,6 +53,7 @@ private IBusControl CreateBus(IServiceProvider serviceProvider)
                     ep.ConfigureConsumer<CreateUserConsumer>(serviceProvider);
                     ep.ConfigureConsumer<DeleteUserConsumer>(serviceProvider);
                     ep.ConfigureConsumer<LoginUserConsumer>(serviceProvider);
+                    ep.ConfigureConsumer<GetUserByIdConsumer>(serviceProvider);

```

Comment:
Yeah, look here. You have 2 consumers here in the same endpoint, receiving same request model, will this code work at all? Have you tested it?

Reply:
Fixed it